docs/sources/setup/install/docker.md

```diff
@@ -9,55 +9,120 @@ weight: 400
 # Install Loki with Docker or Docker Compose
 
 You can install Loki and Promtail with Docker or Docker Compose if you are evaluating, testing, or developing Loki.
-For production, Grafana recommends installing with Tanka or Helm.
+For production, Grafana recommends installing with Helm or Tanka.
 
-The configuration acquired with these installation instructions run Loki as a single binary.
+The configuration files associated with these installation instructions run Loki as a single binary.
 
 ## Prerequisites
 
 - [Docker](https://docs.docker.com/install)
 - [Docker Compose](https://docs.docker.com/compose/install) (optional, only needed for the Docker Compose install method)
 
-## Install with Docker
+## Install with Docker on Linux
 
-**Linux**
+1. Create a directory called `loki`. Make `loki` your current working directory:
 
-Copy and paste the commands below into your command line.
+    ```bash
+    mkdir loki
+    cd loki
+    ```
 
-```bash
-wget https://raw.githubusercontent.com/grafana/loki/v2.9.2/cmd/loki/loki-local-config.yaml -O loki-config.yaml
-docker run --name loki -d -v $(pwd):/mnt/config -p 3100:3100 grafana/loki:2.9.2 -config.file=/mnt/config/loki-config.yaml
-wget https://raw.githubusercontent.com/grafana/loki/v2.9.2/clients/cmd/promtail/promtail-docker-config.yaml -O promtail-config.yaml
-docker run --name promtail -d -v $(pwd):/mnt/config -v /var/log:/var/log --link loki grafana/promtail:2.9.2 -config.file=/mnt/config/promtail-config.yaml
-```
+1. Copy and paste the following commands into your command line to download `loki-local-config.yaml` and `promtail-docker-config.yaml` to your `loki` directory.
+
+    ```bash
+    wget https://raw.githubusercontent.com/grafana/loki/v2.9.2/cmd/loki/loki-local-config.yaml -O loki-config.yaml
```

Comment:
Can we use 3.0.0 here?

Reply:
Updated 2.9.2 => 3.0.0 across the board.